docs/types.rst

```diff
@@ -102,6 +102,10 @@ and to send Ether (in units of wei) to an address using the ``send`` function:
     to make safe Ether transfers, always check the return value of ``send`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+* ``transfer``
```

Comment:
The reason I've left it this way because it was well explained. Need to work on rephrasing the whole if we swap it.